docs/en/mavlink/index.md

```diff
@@ -0,0 +1,88 @@
+# MAVLink Messaging
```

Comment:
FYI, moved the document under mavlink. There is a redirect. This is just so that all core docs are in one place.